src/mappings/colonyNetwork.ts

```diff
@@ -24,8 +26,28 @@ export function handleColonyAdded(event: ColonyAdded): void {
     colony.metadataHistory = []
   }
 
+  let tokenAddress = event.params.token.toHexString()
+  let token = Token.load(tokenAddress)
+  if (token == null){
```

Comment:
We copied this block from [colony.ts](https://github.com/JoinColony/subgraph/blob/master/src/mappings/colony.ts#L56) to get things going - can we refactor so it only appears once?

Reply:
Good point, forgot about it, updated!